Seda/Seda.xcodeproj/project.pbxproj

```diff
@@ -418,7 +422,7 @@
 			buildSettings = {
 				ASSETCATALOG_COMPILER_APPICON_NAME = AppIcon;
 				CODE_SIGN_STYLE = Automatic;
-				DEVELOPMENT_TEAM = 77HZ7RQNQN;
+				DEVELOPMENT_TEAM = 4F7LBHA35G;
```

Comment:
We're either of you guys having issues using your personal development team with the project? I was having trouble getting the app to run on my phone because of this

Reply:
The only issue I had in the beginning was that I needed to agree to new terms of service on my apple dev account.

Reply:
It's saying that I cannot use the bundle identifier "com.djchen.Seda" with my personal development team.

Reply:
not sure about the solution have you tried to change your signing team?
![Screen Shot 2020-05-22 at 2 25 14 PM](https://user-images.githubusercontent.com/61523891/82710727-6718c680-9c38-11ea-8acc-d4998324e72d.png)

Reply:
Yeah, but the bundle identifier is not linked to my signging team.

Reply:
I've tried on a different account and got the error in the image. I fix that by plugin another device, select that device as the target as pressed try again.
![Screen Shot 2020-05-22 at 2 52 52 PM](https://user-images.githubusercontent.com/61523891/82712048-fd022080-9c3b-11ea-8dae-91cdb26cf871.png)

Reply:
I found this image from the internet since the error has gone.

Reply:
I'm getting a different error, but something similar:

Failed to register bundle identifier.
The app identifier "com.djchen.Seda" cannot be registered to your development team because it is not available. Change your bundle identifier to a unique string to try again.